### PR TITLE
Fix vulnerabilities in dependencies

### PR DESCRIPTION
**Pull request by Winggo**
image

- Upgrade postcss to >8.2.10
- Upgrade lodash to >4.17.21
- Upgrade hosted-git-info to >2.8.9
- Upgrade url-parse to >1.5.0

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6005,66 +6041,55 @@ posix-character-classes@^0.1.0:
...
   resolved "https://registry.npmjs.org/postcss-value-parser/-/postcss-value-parser-4.1.0.tgz#443f6a20ced6481a2bda4fa8532a6e55d789a2cb"
   integrity sha512-97DXOFbQJhk71ne5/Mt6cOu6yxsSfM0QGQyl0L25Gca4yGWEGJaig7l7gbCX623VqTBNGLRLaVUCnNkcedlRSQ==
 
-postcss@^7.0.14, postcss@^7.0.5, postcss@^7.0.6:
-  version "7.0.35"
-  resolved "https://registry.npmjs.org/postcss/-/postcss-7.0.35.tgz#d2be00b998f7f211d8a276974079f2e92b970e24"
-  integrity sha512-3QT8bBJeX/S5zKTTjTCIjRF3If4avAT6kqxcASlTWEtAFCb9NH0OUxNDfgZSWdP5fJnBYCMEWkIFfWeugjzYMg==
+postcss@^8.2.10:
+  version "8.2.15"
+  resolved "https://registry.yarnpkg.com/postcss/-/postcss-8.2.15.tgz#9e66ccf07292817d226fc315cbbf9bc148fbca65"
```
Comment (Winggo) [inline yarn.lock:6087]: postcss > 8.2.10

File: yarn.lock [hunk 2]
```diff
@@ -5026,10 +5050,10 @@ lodash.sortby@^4.7.0:
   resolved "https://registry.npmjs.org/lodash.sortby/-/lodash.sortby-4.7.0.tgz#edd14c824e2cc9c1e0b0a1b42bb5210516a42438"
   integrity sha1-7dFMgk4sycHgsKG0K7UhBRakJDg=
 
-lodash@^4.15.0, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.3:
-  version "4.17.20"
-  resolved "https://registry.npmjs.org/lodash/-/lodash-4.17.20.tgz#b44a9b6297bcb698f1c51a3545a2b3b368d59c52"
-  integrity sha512-PlhdFcillOINfeV7Ni6oF1TAEayyZBoZ8bcshTHqOYJYlrqzRK5hagpagky5o4HfCzzd1TRkXPMFq6cKk9rGmA==
+lodash@4.17.21, lodash@^4.15.0, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.3:
+  version "4.17.21"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.21.tgz#679591c564c3bffaae8454cf0b3df370c3d6911c"
```
Comment (Winggo) [inline yarn.lock:5055]: lodash > 4.17.21

File: yarn.lock [hunk 3]
```diff
@@ -4098,10 +4123,12 @@ homedir-polyfill@^1.0.1:
   dependencies:
     parse-passwd "^1.0.0"
 
-hosted-git-info@^2.1.4:
-  version "2.8.8"
-  resolved "https://registry.npmjs.org/hosted-git-info/-/hosted-git-info-2.8.8.tgz#7539bd4bc1e0e0a895815a2e0262420b12858488"
-  integrity sha512-f/wzC2QaWBs7t9IYqB4T3sR1xviIViXJRJTWBlx2Gf3g0Xi5vI7Yy4koXQ1c9OYDGHN9sBy1DQ2AB8fqZBWhUg==
+hosted-git-info@4.0.2, hosted-git-info@^2.1.4:
+  version "4.0.2"
+  resolved "https://registry.yarnpkg.com/hosted-git-info/-/hosted-git-info-4.0.2.tgz#5e425507eede4fea846b7262f0838456c4209961"
```
Comment (Winggo) [inline yarn.lock:4128]: hosted-git-info > 2.8.9

File: yarn.lock [hunk 4]
```diff
@@ -7778,10 +7814,10 @@ url-parse-lax@^3.0.0:
   dependencies:
     prepend-http "^2.0.0"
 
-url-parse@^1.4.3:
-  version "1.4.7"
-  resolved "https://registry.npmjs.org/url-parse/-/url-parse-1.4.7.tgz#a8a83535e8c00a316e403a5db4ac1b9b853ae278"
-  integrity sha512-d3uaVyzDB9tQoSXFvuSUNFibTd9zxd2bkVrDRvF5TmvWWQwqE4lgYJ5m+x1DbecWkw+LK4RNl2CU1hHuOKPVlg==
+url-parse@1.5.1, url-parse@^1.4.3:
+  version "1.5.1"
+  resolved "https://registry.yarnpkg.com/url-parse/-/url-parse-1.5.1.tgz#d5fa9890af8a5e1f274a2c98376510f6425f6e3b"
```
Comment (Winggo) [inline yarn.lock:7819]: url-parse > 1.5.0

File: webpack.config.js [hunk 5]
```diff
@@ -42,6 +50,7 @@ module.exports = {
             options: {
               importLoaders: 1,
               modules: true,
+              esModule: false,
             },
           },
```
Comment (Winggo) [inline webpack.config.js:55]: Need to add `esModule: false` option to upgraded `css-loader`
https://github.com/vuejs/vue-style-loader/issues/46#issuecomment-670624576

File: package.json [hunk 6]
```diff
@@ -84,5 +84,10 @@
     "webpack": "^4.5.0",
     "webpack-cli": "^3.0.8",
     "webpack-dev-server": "^3.1.3"
+  },
+  "resolutions": {
+    "lodash": "4.17.21",
+    "hosted-git-info": "4.0.2",
+    "url-parse": "1.5.1"
```
Comment (ralbayaty) [inline package.json:91]: instead of exact version resolution, can we make these minor version match with the caret range specifier `^`?

**Review by ralbayaty — APPROVED** [review]
maybe let's try using minor version matching?

File: package.json (hunk 6 above)
Comment (Winggo) [reply package.json:91]: Ah yes. Did not know adding `^` before the versioning is called minor versioning matching. Learned something new today

File: package.json [hunk 7]
```diff
@@ -86,8 +86,8 @@
     "webpack-dev-server": "^3.1.3"
   },
   "resolutions": {
-    "lodash": "4.17.21",
-    "hosted-git-info": "4.0.2",
-    "url-parse": "1.5.1"
+    "lodash": "^4.17.21",
+    "hosted-git-info": "^4.0.2",
+    "url-parse": "^1.5.1"
   }
```
Comment (Winggo) [inline package.json]: This should do the trick?

Comment (ralbayaty) [reply package.json]: blast the node_modules then try it out locally :)

Comment (Winggo) [reply package.json]: Blasted and tested✅ 
Works as expected

**Pull request merged**